experiment_scheduler/master/master.py

```diff
@@ -292,6 +298,40 @@ def _wrap_by_task_status(self, task_id, status):
             status=status,
         )
 
+    def edit_task(self, request, context):
+        task_id = request.task_id
+        cmd = request.cmd
+        if task_id in self.queued_tasks.keys():
+            self.queued_tasks[task_id].command = cmd
+            response = MasterResponse(
+                experiment_id="0", response=MasterResponse.ResponseStatus.SUCCESS
+            )
+        elif task_id in dict(self.running_tasks).keys():
+            response = self.process_monitor.kill_task(
+                self.running_tasks[task_id]["task_manager"], task_id
+            )
+            if response.status == TaskStatus.Status.KILLED:
+                del self.running_tasks[task_id]
+
+            task_name = task_id.split("-")[0]
+            command = cmd
+            task_env = dict()
+            self.queued_tasks[task_id] = MasterTaskStatement(
+                name=task_name, command=command, task_env=task_env
+            )
+            response = MasterResponse(
+                experiment_id="0",
+                response=MasterResponse.ResponseStatus.SUCCESS,  # pylint: disable=E1101
```

Comment:
if edit target task is running, kill its task and put changed task to queued_task.
However, task_env does not served properly. To make queued_task instance with running_tasks, running task must have queued_task's member values.